modules/publication/ajax/FileUpload.php

```diff
@@ -86,7 +88,7 @@ function uploadPublication()
         $leadInvID = $db->getLastInsertId();
     }
     if (!isset($desc) || !isset($leadInvest) || !isset($leadInvestEmail)) {
```

Comment:
You can pass multiple variables in `isset`:

```php
if (!isset($desc, $leadInvest, $leadInvestEmail)) {
```

Reply:
@johnsaigle TIL